src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1475,6 +1477,8 @@ namespace DataHeatBalance {
         Real64 OABalanceFanElec = 0.0;       // Fan Electricity {W} due to OA air balance
         Real64 SumEnthalpyM = 0.0;           // Zone sum of EnthalpyM
         Real64 SumEnthalpyH = 0.0;           // Zone sum of EnthalpyH
+        // reporting flags
+        bool ReportWBGT = true; // whether the wetbulb globe temperature is reqeusted as an output variable or used as an EMS sensor
```

Comment:
Sorry - it's this flag that should be initialized to false.

Reply:
Oh! right right. This should be false.